debian/libostree-1-1.symbols

```diff
@@ -148,6 +148,7 @@ libostree-1.so.1 libostree-1-1 #MINVER#
  ostree_remote_ref@LIBOSTREE_2017.6_EXPERIMENTAL 2017.8
  ostree_remote_unref@LIBOSTREE_2017.6_EXPERIMENTAL 2017.8
  ostree_remote_get_name@LIBOSTREE_2017.7_EXPERIMENTAL 2017.8
+ ostree_remote_get_type@LIBOSTREE_2017.14_EXPERIMENTAL 2017.12-0endless3
```

Comment:
I think you also need the `LIBOSTREE_2017.14_EXPERIMENTAL` at the top of this file.